README.md

```diff
@@ -1,42 +1,37 @@
-# PyDPF-Post - Ansys Data PostProcessing Framework
+# PyDPF-Post
 [![PyAnsys](https://img.shields.io/badge/Py-Ansys-ffc107.svg?logo=data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAABAAAAAQCAIAAACQkWg2AAABDklEQVQ4jWNgoDfg5mD8vE7q/3bpVyskbW0sMRUwofHD7Dh5OBkZGBgW7/3W2tZpa2tLQEOyOzeEsfumlK2tbVpaGj4N6jIs1lpsDAwMJ278sveMY2BgCA0NFRISwqkhyQ1q/Nyd3zg4OBgYGNjZ2ePi4rB5loGBhZnhxTLJ/9ulv26Q4uVk1NXV/f///////69du4Zdg78lx//t0v+3S88rFISInD59GqIH2esIJ8G9O2/XVwhjzpw5EAam1xkkBJn/bJX+v1365hxxuCAfH9+3b9/+////48cPuNehNsS7cDEzMTAwMMzb+Q2u4dOnT2vWrMHu9ZtzxP9vl/69RVpCkBlZ3N7enoDXBwEAAA+YYitOilMVAAAAAElFTkSuQmCC)](https://docs.pyansys.com/)
 [![Python](https://img.shields.io/pypi/pyversions/ansys-dpf-post?logo=pypi)](https://pypi.org/project/ansys-dpf-post/)
 [![pypi](https://badge.fury.io/py/ansys-dpf-post.svg?logo=python&logoColor=white)](https://pypi.org/project/ansys-dpf-post)
 [![MIT](https://img.shields.io/badge/License-MIT-yellow.svg)](https://opensource.org/licenses/MIT)
 
-Ansys Data Processing Framework (DPF) provides numerical simulation 
-users and engineers with a toolbox for accessing and transforming simulation 
-data. With DPF, you can perform complex preprocessing or postprocessing of
-large amounts of simulation data within a simulation workflow.
+PyDPF-Post is part of the Ansys Data Processing Framework (DPF), which provides reusable operators
+that let you access and transform simulation data coming from different Ansys and third-party
+result files.
 
-The Python `ansys-dpf-post` package provides a high-level, physics-oriented
```

Comment:
@PipKat the part about the library being dedicated to high-level, physics-oriented postprocessing is important as this explains the goal of the library and why it is different from PyDPF-Core.
A bit more info: PyDPF-Core is physics-agnostic and operator-oriented and a direct exposure of the DPF features and objects, with a few helpers, and not restricted to postprocessing.
PyDPF-Post on the other hand was made with the intention of providing a more pythonic and user-friendly API dedicated to postprocessing, with new objects meant to provide a specific interface for each simulation domain (mechanics vs fluids vs...), and most importantly no need to manipulate operators or workflows. The concepts do not appear.

Reply:
@PProfizi I added the differences you supplied to both the README and the doc landing page.